src/sass/components/Header.scss

```diff
@@ -0,0 +1,12 @@
+.Header {
```

Comment:
Turning these off renders nearly (if not completely) identical ... or am I missing something?